public/scripts/components/pages/Stories.js

```diff
@@ -1,11 +1,74 @@
 import React from 'react';
+import $ from 'jquery';
+import Stories from './../../collections/StoryCollection';
+import PicTitleThumb from './../sub-components/pic-title-thumb.js';
 
-export default React.createClass({
-	render: function() {
-		return (
+
+const StoryList = React.createClass ({
+	getInitialState: function(){
+		//always returns an object
+		return{Stories:[]};
```

Comment:
You should set the initial Stories property to be Stories rather than an empty array, in case for some reason the stories are already fetched